jerry-core/ecma/base/ecma-helpers.c

```diff
@@ -1269,31 +1231,31 @@ ecma_set_property_configurable_attr (ecma_property_t *prop_p, /**< property */
  * @return true / false
  */
 inline bool __attr_always_inline___
-ecma_is_property_lcached (ecma_property_t *prop_p) /**< property */
+ecma_is_property_lcached (ecma_property_t *property_p) /**< property */
```

Comment:
I didn't change this function because lcache is a very special flag used only by the lcache, and the property pointer is always available. So it is not a general function like the others.